app/controllers/api/internal/v1/defendants_controller.rb

```diff
@@ -46,6 +46,16 @@ def allowed_params
         def transformed_params
           @transformed_params ||= unlink_params.slice(*allowed_params).to_hash.transform_keys(&:to_sym).merge(defendant_id: params[:id])
         end
+
+        def serialization_options
+          return { include: inclusions, params: { inclusions: inclusions } } if params[:include].present?
```

Comment:
Not sure if we need this fine grained control over sending the params inside.
The reason this has been added to case search is  _if_ the UI requests a hearing to be included (additional calls need to be made then)
I assume this would have worked out of the box for your use case?

Reply:
removed that now